packages/blocks/src/api/registration.js

```diff
@@ -44,12 +44,15 @@ import { isValidIcon, normalizeIconObject } from './utils';
 let serverSideBlockDefinitions = {};
 
 /**
- * Set the server side block definition of blocks.
+ * Sets the server side block definition of blocks.
  *
  * @param {Object} definitions Server-side block definitions
  */
 export function unstable__bootstrapServerSideBlockDefinitions( definitions ) { // eslint-disable-line camelcase
```

Comment:
Doesn't need to be done here, but... we should fix this function name to adhere to standards (`__unstable`, not `unstable__`).

Reply:
Yes, I noticed it as well. I can include in this PR as this is straightforward change.

Reply:
WordPress core uses `unstable__bootstrapServerSideBlockDefinitions`. Let's leave it as is for now.

Reply:
> WordPress core uses `unstable__bootstrapServerSideBlockDefinitions`. Let's leave it as is for now.

Let's have a plan for following-up. Ideally if there's more effort in changing it, we try to work toward just not needing it at all. Would that be considered part of the long-term effort of the block manifest?

I seem to recall some prior thoughts or conversations around allowing `registerBlockType` to be called multiple times to additively override settings, or to have separate interfaces for _registering_ and _extending_/_implementing_ block types.

Reply:
> Let's have a plan for following-up. Ideally if there's more effort in changing it, we try to work toward just not needing it at all. Would that be considered part of the long-term effort of the block manifest?

It seems like it would be not that easy to align having plugins using a new name while WordPress core still uses an old one. We could mitigate it by enforcing that plugin is installed only with a version of WP core which uses the updated name but I guess this might be not that easy to get there without a transition period. I think we should start talking to REST API team on how to expose the same data `unstable__bootstrapServerSideBlockDefinitions` exposes from the server to be reachable through API endpoint. What do you think about opening a follow-up issue with some details explained and reaching out to REST API team as next step?